README.md

```diff
@@ -33,16 +37,56 @@ const layoutEngine = constructLayoutEngine({routes, applications})
 start()
 ```
 
-
 ### Constructing Routes
+
 single-spa-layout supports two different sytanxes defining applications and routes at this time:
+
 - [JSX](#JSX) (compiles to JSON syntax)
```

Comment:
I'm thinking we'll remove JSX in favor of HTML sent down in the HTML file. I think JSX will cause confusion in the following ways:

1. Can I call setState?
2. Is this react?
3. `className` vs `class`

Reply:
Yeah I can do that in the next MR